pyhf/__init__.py

```diff
@@ -1,4 +1,6 @@
 from . import tensor, optimize
+from .version import __version__
+assert __version__  # Used to satisfy pyflakes
```

Comment:
remove the assert and add `__all__ = ['__version__']` which should also make pyflakes happy, but is more correct.